programs/zk-token-proof/benches/verify_proofs.rs

```diff
@@ -95,7 +96,42 @@ fn bench_grouped_ciphertext_validity(c: &mut Criterion) {
     )
     .unwrap();
 
-    c.bench_function("grouped_ciphertext_validity", |b| {
+    c.bench_function("grouped_ciphertext_2_handles_validity", |b| {
+        b.iter(|| {
+            proof_data.verify_proof().unwrap();
+        })
+    });
+}
+
+fn bench_grouped_ciphertext_3_handles_validity(c: &mut Criterion) {
+    let source_keypair = ElGamalKeypair::new_rand();
+    let source_pubkey = source_keypair.pubkey();
+
+    let destination_keypair = ElGamalKeypair::new_rand();
+    let destination_pubkey = destination_keypair.pubkey();
+
+    let auditor_keypair = ElGamalKeypair::new_rand();
+    let auditor_pubkey = auditor_keypair.pubkey();
+
+    let amount: u64 = 55;
```

Comment:
Yep it is inconsequential since the proof only certifies that a ciphertext is well-formed. If it were, then the proof would leak information about the encrypted amount since anyone can try verifying the proof and measure the time!